advocacy_docs/partner_docs/EsriArcGISProandArcGISEnterprise/04-ConfiguringEsriArcGISProandEsriArcGISEnterprise.mdx

```diff
@@ -0,0 +1,179 @@
+---
+title: 'Configuring Esri ArcGIS Pro and Esri ArcGIS Enterprise'
+description: 'Walkthrough on configuring Esri ArcGIS Pro and Esri ArcGIS Enterprise'
+---
+
+Implementing Esri ArcGIS Pro with EDB Postgres Advanced Server requires the following components:
+
+- A running EDB Postgres Advanced Server instance
+- A current version of PostGIS installed on EDB Postgres Advanced Server
+- GIS data set
+- Esri License files
```

Comment:
Don't need to say this.
Better to just say a running Esri Enterprise Server or would it be better to say
running Esri Enterprise environment

Reply:
If we are doing two guides I think for the ArcGIS Pro guide we could say "A running Esri Enterprise Server" and for the Enterprise Guide we could say a "Running Esri Enterprise Environment"

Reply:
Looking at this now on the Pro side I don't think we need the server piece in there at all.  We really just need active licensing which should be established when a customer is working on setting up their Esri environments with their tech support.  

On the Enterprise side I would put "A running Esri Enterprise Environment" for the pre-reqs because that is the big piece that needs to have all of the separate pieces running before they can do any Sharing.